Loop Simple Calculator.py

```diff
@@ -8,37 +8,27 @@ def simpcalc() :
 	op = input("Enter an operator: ")
 	
 	num2 = float(input("Enter your second number: "))
-		
-#Defining operators
-	addition = 1
-	subtraction = 1
-	division = 1
-	multiplication = 1
-	
 
 	if op == "+" :
-		addition = num1 + num2
-		print(addition)
+		result = num1 + num2
 		
 	elif op == "-" :
-		subtraction = num1 - num2
-		print(subtraction) 
+		result = num1 - num2
 		
 	elif op == "/" :
-		if num1 or num2 == 0:
-			num1 = int(num1)
-			num2 = int(num2)
-			division = num1 / num2
+		if num2 != 0 :
+			result = num1 / num2
 		else:
-			division = num1 / num2
-			print(division)
+			result = "Can't divide by 0"
```

Comment:
(num1 or num2 == 0) really means ((num1) or (num2 == 0)) so this will be always true if num1 is true(in case of numbers, that means non-zero) so you couldn't perform any division except for (zero/nonzero) which is not really helpful.